pyvecorg/data/projects.yml

```diff
@@ -177,7 +177,7 @@ note:
   text:
     cs: |-
       Pyvec nikomu **neříká, co má dělat**. [Pomáhá těm, kteří se o něco snaží](https://docs.pyvec.org/operations/support.html)
-      &mdash; je to **servisní organizace**. Založení srazu ve tvém městě? Kurz
+      &mdash; je to nezisková **servisní organizace**. Založení srazu ve tvém městě? Kurz
```

Comment:
Tuhle změnu by asi bylo dobry udělat teda i o radek níž v anglické verzi a přidat tam to non profit

Reply:
@honzajavorek Máš pravdu, díky za všimnutí! PR: https://github.com/pyvec/pyvec.org/pull/225